spec/links_spec.rb

```diff
@@ -1,19 +1,19 @@
-require_relative '../lib/parser'
+require_relative '../lib/awesome_list'
 require 'net/http'
 require 'faraday'
 
 describe 'links' do
   let(:readme) { File.open("README.md").read }
-  let(:list) { AwesomeListRender.parse(readme) }
+  let(:list) { AwesomeList.parse(readme) }
 
   it 'must not appear twice' do
-    list_links = list.project_links.map { |l| l.map{|ll| ll['href'] }}.flatten
-    expect(list_links).to eq(list_links.uniq), duplicated_links_error(list_links)
+    links = list.projects.css('li a').map { |a| a['href'] }
```

Comment:
Since we're repeating this line, maybe move it outside so it's common or move it to a function?